LDRH_choice.py

```diff
@@ -231,7 +247,14 @@ def importConditions(path):
         'Spatial': data.StairHandler(startVal = 150,
             stepType = 'db', stepSizes=[3,3,2,2,1,1],#[8,4,4,2,2,1,1], #reduce step size every two reversals
             minVal=0, maxVal=350, nUp=1, nDown=3,  #will home in on the 80% threshold
-            nTrials = 10)
+            nTrials = 10),
+        'posterior_matching': {
+            'Math': handler_function.posterior_matching(startval = math_max , minval = [0,0,0,0], maxval = math_max, axis = 4),
```

Comment:
Let's be consistent with our keyword args and make them match the handlers above.

```startVal``` not ```startval```.

```minVal``` and ```maxVal``` too.

Also, seeing how this gets used later, we shouldn't even conditionally define them, it should be either or.

If we are using posterior_matching, we should use the posterior_matching handlers directly in all_handlers, if not, we use the usual ones.